CMakeLists.txt

```diff
@@ -88,6 +88,12 @@ else()
 	message(STATUS "SPIRV-Cross: Git not found, using unknown build version.")
 endif()
 
+find_program(CCACHE_FOUND ccache)
+if (CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
```

Comment:
After the `find_program()` above, the variable `CCACHE_FOUND` will contain the full pathname of the `ccache` program. You should probably use it, instead of assuming that `ccache` will be in the user's `PATH`.